src/Header/Header.css

```diff
@@ -0,0 +1,4 @@
+.app > .header {
+	border-radius: var(--app-padding);
+	padding: var(--app-half-padding) 0;
```

Comment:
Does this work? Since neither of those CSS vars exist in this file, I'd be impressed if they still worked

You'll probably need to copy these lines from the App CSS file as well -

```
:root {
	--app-padding: 1rem;
	--app-half-padding: calc(var(--app-padding) / 2);
}
```

Reply:
Or as per my comment above, just empty the CSS file (provided that doesn't look horrible)